pybossa/view/projects.py

```diff
@@ -1722,7 +1824,7 @@ def bulk_redundancy_update(short_name):
             })
 
         else:
-            args = parse_tasks_browse_args(request.json.get('filters'))
+            args = parse_tasks_browse_args(req_data.get('filters', ''))
```

Comment:
Similar comment and [solution](https://github.com/bloomberg/pybossa/pull/655/files#r771642803) as above to check for None.

Reply:
thanks Kory, I got it working!